web/app.js

```diff
@@ -164,6 +165,9 @@ let PDFViewerApplication = {
     });
 
     this.initialized = true;
+
+    // update url on scroll
+    setupHashUpdating();
```

Comment:
It feels like this is part of the initialization, so I think it should be moved above `this.initialized = true` above.